### ISSUE TITLE
lvg fails when vg_options is not supplied

### ISSUE THREAD

**Issue by hughsaunders** [ansible/ansible-modules-extras#80]
##### Steps To Reproduce:

```
ansible -i inventory host -m lvg -a "vg='cinder' pvs='/dev/sda5'"
```
##### Actual Results:

Volume group is not created as the module attempts to split on a parameter that hasn't been supplied, note that <a href="http://docs.ansible.com/lvg_module.html">the docs</a> specify this parameter as optional. 

```
hugh@host:~/$ ansible -i inventory node18 -m lvg -a "vg='cinder' pvs='/dev/sda5'"
node18 | FAILED >> {
    "failed": true,
    "msg": "Traceback (most recent call last):\n  File \"<stdin>\", line 1458, in <module>\n  File \"<stdin>\", line 119, in main\nAttributeError: 'NoneType' object has no attribute 'split'\n",
    "parsed": false
}
```

**Comment by srgvg** [comment]
This issue was already solved, if not in 1.7 at least in 1.8-devel, by adding a defaults options to that parameter.

https://github.com/ansible/ansible-modules-extras/commit/867702b26a14bbf03da6799c69f74e8f65a66b33

**Comment by hughsaunders** [comment]
@srvg thanks, the perils of old code!
